script/DepositScript.s.sol

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.19;
+
+import "forge-std/Script.sol";
+import {ObolValidatorManager} from "src/ovm/ObolValidatorManager.sol";
+
+//
+// This script calls deposit() on a ObolValidatorManager contract.
+// To run this script, the following environment variables must be set:
+// - PRIVATE_KEY: the private key of the account that will deploy the contract
+// Example usage:
+//   forge script script/DepositScript.s.sol --sig "run(address)" \
+//   --rpc-url https://rpc.pectra-devnet-5.ethpandaops.io/ --broadcast "<ovm_address>"
+//
+contract DepositScript is Script {
+    function run(address ovmAddress) external {
+        uint256 privKey = vm.envUint("PRIVATE_KEY");
+
+        vm.startBroadcast(privKey);
+
+        ObolValidatorManager ovm = ObolValidatorManager(payable(ovmAddress));
+
+        // Update these values from the deposit_data json file
```

Comment:
Fine for now, we can make a fancier version that parses json some other time :)

Reply:
Sure we must invest into such the tooling!